src/styles/main.scss

```diff
@@ -1,3 +1,48 @@
+@import './variables.scss';
+
 body {
-  margin: 0;
+  margin: 0 auto;
+  font-family: $font-family;
+  font-size: $font-size;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $cell-size;
+  gap: $gap-size;
+
+  &__day {
+    height: $height;
+    width: $width;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    border: 1px solid #000;
+    box-sizing: border-box;
+
+    &:hover {
+      background-color: pink;
+      cursor: pointer;
+    }
+
+    @for $number from 1 through 29 {
+      &:nth-child(#{$number})::before {
+        content: "#{$number}";
+      }
+    }
```

Comment:
Why do you have 2 different cycles for setting content(from 1 to 29 and from 28 to 31)?